### ISSUE TITLE
Error on Excel import

### ISSUE THREAD

**Issue by jamesrwelch** [MauroDataMapper-Plugins/mdm-plugin-excel#3]
See attached test file.
Finalised existing model, imported, made some changes, and then tried to upload new version.

```
"Property [modelName] of class [class uk.ac.ox.softeng.maurodatamapper.plugins.excel.datamodel.provider.importer.parameters.ExcelDataModelFileImporterProviderServiceParameters] cannot be null
```
This is using the 'import as new documentation version' option.

[trak_lab_results_numeric_20210624T082905_enumeration.xlsx](https://github.com/MauroDataMapper-Plugins/mdm-plugin-excel/files/6709366/trak_lab_results_numeric_20210624T082905_enumeration.xlsx)

If no obvious other solution found, it might be related to the old model having a different authority from the current default.

**Comment by oughnic** [comment]
I can confirm this behaviour with a different test data model, exported from mauro (home instance) and then reimported as a branch
``` 
Import Errors (1)
Property [modelName] of class [class uk.ac.ox.softeng.maurodatamapper.plugins.excel.datamodel.provider.importer.parameters.ExcelDataModelFileImporterProviderServiceParameters] cannot be null
```
![image](https://user-images.githubusercontent.com/27196895/123410418-388dbd80-d5a7-11eb-8f1d-8e02dca5af7b.png)

**Comment by jamesrwelch** [comment]
This is because the 'modelName' parameter has been annotated as 'hidden', but this loses the annotation of 'optional', making it mandatory. Unless someone has a good reason, I'll update and release a new version

**Comment by jamesrwelch** [comment]
Fixed with release `4.0.1` (forgot to tag this issue in commit)
